proto/self_serve/orb/v1/orb.proto

```diff
@@ -11,12 +11,14 @@ message AnnounceOrbId {
   bool is_self_serve_enabled = 2;
 }
 message CaptureStarted {}
+message StartCaptureTimeout {}
```

Comment:
```suggestion
message CaptureTriggerTimeout {}
```

Reply:
I agree, I just copied the corresponding message from `app` and put a Timeout next to it 🙈